git-integration/README.md

```diff
@@ -0,0 +1,8 @@
+# Roadmap for Git Integration
+
+Many notebook users utilize git and GitHub to manage versions of
+their notebooks and share them with others. Sometimes this process
+can be quite difficult as creating diffs of the notebook format
+comes with a lot of challenges. We are hoping to address this
+issue and create tightly-coupled git integration in the nteract
+desktop application.
```

Comment:
I'd distill this into these bullet points:

* Ability to commit straight from the notebook interface
* Visual representation of diffs

We can't solve the issue of the notebooks themselves being difficult to handle, luckily the diffing tools are being worked on by others including @minrk.